api/envoy/extensions/filters/http/json_to_metadata/v3/json_to_metadata.proto

```diff
@@ -112,5 +112,8 @@ message JsonToMetadata {
   }
 
   // Rules to match json body of requests
-  MatchRules request_rules = 1 [(validate.rules).message = {required: true}];
+  MatchRules request_rules = 1;
+
+  // Rules to match json body of responses
+  MatchRules response_rules = 2;
```

Comment:
Please add a clarification that at least one of `request_rules` and `response_rules` must be provided.